test-butler-library/src/main/java/com/linkedin/android/testbutler/TestButler.java

```diff
@@ -158,6 +158,22 @@ public static void setWifiState(boolean enabled) {
     }
 
     /**
+     * Enable/disable the Wifi connection on the emulator
```

Comment:
Might also be nice to call out when this is useful instead of just toggling airplane mode (which is another feature we want to add to Test Butler)